spring-cloud-config-client/src/test/java/org/springframework/cloud/config/client/ConfigServerConfigDataLocationResolverTests.java

```diff
@@ -167,22 +167,6 @@ void urisInLocationOverridesProperty() {
 		assertThat(resource.getProperties().getUri()).containsExactly(locationUri.split(","));
 	}
 
-	@Test
```

Comment:
We are no longer using the properties stored in the bootstrap context, this test is not valid